frontend/packages/kubevirt-plugin/console-extensions.json

```diff
@@ -216,7 +222,7 @@
       }
     },
     "flags": {
-      "required": ["KUBEVIRT"]
+      "required": ["KUBEVIRT", "KUBEVIRT_IS_CLUSTER_ADMIN"]
```

Comment:
```suggestion
      "required": ["KUBEVIRT", FLAGS.CAN_GET_NS, FLAGS.OPENSHIFT]
```

Reply:
Could you please explain why you're suggesting this change?

Reply:
This is the flags used to hide the cluster overview [1], can you try and see if it works for us too ?

[1] [admin-nav.tsx](https://github.com/openshift/console/blob/291ccda04403e35dff923fb4217ef4882f1179c1/frontend/public/components/nav/admin-nav.tsx#L127)

Reply:
That works, so I've updated it. It does bother me that the intention isn't clear, though. I left the cleanup for `getOSImagesNS`, though.